### PR TITLE
Update known-issues.md

### PR DESCRIPTION
**Pull request by abraunegg**
* Add 'SSL_ERROR_SYSCALL, errno 104' as known issue

### CODE REVIEW

**Comment by abraunegg** [comment]
@norbusan 
if you can check for readability, modify if needed

File: docs/known-issues.md [hunk 1]
```diff
@@ -32,3 +32,44 @@ Technically, the client is 'working' correctly, as, when moving files, you are '
...
+
+**Issue Tracker:** https://github.com/abraunegg/onedrive/issues/884
+
+**Description:**
+
+When running the client and performing an upload or download operation, the application just stops working without any reason or explanation.
+
+**Explanation:**
+
+The client is heavilly dependant on Curl and OpenSSL to perform the activities with the Microsoft OneDrive service. Generally, when this issue occurs, the following is found in the HTTPS Debug Log:
```
Comment (norbusan) [inline docs/known-issues.md:51]: heavily?

File: docs/known-issues.md [hunk 2]
```diff
@@ -32,3 +32,44 @@ Technically, the client is 'working' correctly, as, when moving files, you are '
...
+--verbose --verbose --debug-https
+```
+
+This is indicative of the following:
+* Some sort of flaky Internet connection somewhere between you and the OneDrive service
+* Some sort of 'broken' HTTPS transparent inspection service inspecting your traffic somewhere between you and the OneDrive service
+
+**How to resolve:**
+
+The best avenue's of action here are:
```
Comment (norbusan) [inline docs/known-issues.md:66]: avenue of action

File: docs/known-issues.md [hunk 3]
```diff
@@ -32,3 +32,44 @@ Technically, the client is 'working' correctly, as, when moving files, you are '
...
+* Ensure your OS is as up-to-date as possible
+* Get support from your OS vendor
+* Speak to your ISP or Help Desk for assistance
+* Open a ticket with OpenSSL and/or Curl teams to better handle this sort of connection failure
+* Generate a HTTPS Debug Log for this application and open a new support request with Microsoft and provide the debug log file for their analysis.
+
+If you wish to diagnosing this issue further, refer to the following:
```
Comment (norbusan) [inline docs/known-issues.md:73]: to diagnose

**Pull request merged**